.prettierrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   trailingComma: 'all',
   semi: false,
   printWidth: 180,
-}
+  tabWidth: 2,
```

Comment:
Why did you add this? It's the default value, shouldn't be necesary

Reply:
had issues with the ide not picking up previously. works now, will remove.